config/main.py

```diff
@@ -2173,8 +2175,8 @@ def add_portchannel(ctx, portchannel_name, min_links, fallback, fast_rate):
     db = ValidatedConfigDBConnector(ctx.obj['db'])
     if ADHOC_VALIDATION:
         if is_portchannel_name_valid(portchannel_name) != True:
-            ctx.fail("{} is invalid!, name should have prefix '{}' and suffix '{}'"
-                    .format(portchannel_name, CFG_PORTCHANNEL_PREFIX, CFG_PORTCHANNEL_NO))
+            ctx.fail("{} is invalid!, name should have prefix '{}' and suffix '{}' and its length should not exceed {} characters"
+                    .format(portchannel_name, CFG_PORTCHANNEL_PREFIX, CFG_PORTCHANNEL_NO, CFG_PORTCHANNEL_NAME_TOTAL_LEN_MAX))
```

Comment:
@arfeigin what if `CFG_PORTCHANNEL_NAME_TOTAL_LEN_MAX` will be set to some other unexpected value? Do we need to add a validation to `is_portchannel_name_valid` to check the `IFNAMSIZ` explicitly?

Reply:
Good catch, I removed this variable is it used in the same manner IFNAMSIZ is. Now it will check for IFNAMSIZ - thing is someone can change this as well in the future. Is there a way to import it as done in cpp? (from net/if)